### PR TITLE
[Notifications] New notifications available

### PR DESCRIPTION
**Pull request by marciocloudflare**

### CODE REVIEW

File: products/fundamentals/src/content/notifications/notification-available.md [hunk 1]
```diff
@@ -22,78 +22,124 @@ Each Notification carries different types of information about the status of you
...
+
+Customers who are using Cloudflare for Secondary DNS and want to receive notifications about failure or success of zone transfers from their primary nameservers.
+
+**Included with**
+
+Enterprise plans.
+
+**What should you do if you receive one?**
+
+Success alerts require no further action. Actions for notifications regarding failures will depend on the type of failure. Possible actions include:
```
Comment (pedrosousa) [inline products/fundamentals/src/content/notifications/notification-available.md]:
```suggestion
Success alerts require no further action. Actions for failure notifications will depend on the type of failure. Possible actions include: 
```

File: products/fundamentals/src/content/notifications/notification-available.md [hunk 2]
```diff
@@ -22,78 +22,124 @@ Each Notification carries different types of information about the status of you
...
+
+Enterprise plans.
+
+**What should you do if you receive one?**
+
+Success alerts require no further action. Actions for notifications regarding failures will depend on the type of failure. Possible actions include: 
+
+1. Checking the Access Control List (ACL) on your primary nameserver
+1. Checking if Cloudflare IPs have been [configured correctly on your primary nameserver](https://support.cloudflare.com/hc/articles/360001356152#access-control-list-configuration)
+1. Checking logs on primary nameservers for other errors.
```
Comment (pedrosousa) [inline products/fundamentals/src/content/notifications/notification-available.md]:
```suggestion
* Checking the Access Control List (ACL) on your primary nameserver.
* Checking if Cloudflare IPs have been [configured correctly on your primary nameserver](https://support.cloudflare.com/hc/articles/360001356152#access-control-list-configuration).
* Checking logs on primary nameservers for other errors.
```

File: products/fundamentals/src/content/notifications/notification-available.md [hunk 3]
```diff
@@ -137,62 +183,60 @@ Purchase of Magic Transit and/or BYOIP.
...
 
 <details>
-<summary>Flow-based Monitoring: Volumetric Attack</summary>
+<summary>Pool Enablement</summary>
 <div>
 
 **Who is it for?**
 
-Magic Transit On Demand customers who are using Flow Based Monitoring to detect attacks when Magic Transit is disabled.
+Customers who want to be warned about the enable/disable status of their pools.
```
Comment (pedrosousa) [inline products/fundamentals/src/content/notifications/notification-available.md]:
```suggestion
Customers who want to be warned about status changes (enabled/disabled) in their pools.
```

File: products/fundamentals/src/content/notifications/notification-available.md [hunk 4]
```diff
@@ -357,81 +419,120 @@ Enterprise plans with paid add-on.
...
 
 <details>
-<summary>Secondary DNS all Primaries Failing</summary>
+<summary>Stream Live Notifications</summary>
 <div>
 
 **Who is it for?**
 
-Enterprise customers who have at least one secondary zone in their account and want to receive a notification if all of their primary nameservers are failing.
+Customers who are using Stream and want to receive webhooks with status of their videos.
```
Comment (pedrosousa) [inline products/fundamentals/src/content/notifications/notification-available.md]:
```suggestion
Customers who are using Stream and want to receive webhooks with the status of their videos.
```

File: products/fundamentals/src/content/notifications/notification-available.md [hunk 5]
```diff
@@ -357,81 +419,120 @@ Enterprise plans with paid add-on.
...
 
 **Included with**
 
-Purchase of Secondary DNS.
+Stream Subscription.
```
Comment (pedrosousa) [inline products/fundamentals/src/content/notifications/notification-available.md]:
```suggestion
Stream subscription.
```

File: products/fundamentals/src/content/notifications/notification-available.md [hunk 6]
```diff
@@ -357,81 +419,120 @@ Enterprise plans with paid add-on.
...
+<summary>Workers Usage Report</summary>
+<div>
+
+**Who is it for?**
+
+Developers using Workers, especially those on the Unbound usage model.
+
+**Included with**
+
+Workers Subscription (free or paid).
```
Comment (pedrosousa) [inline products/fundamentals/src/content/notifications/notification-available.md]:
```suggestion
Workers subscription (free or paid).
```

File: products/fundamentals/src/content/notifications/notification-available.md [hunk 7]
```diff
@@ -357,81 +419,120 @@ Enterprise plans with paid add-on.
...
+<summary>Workers Weekly Summary</summary>
+<div>
+
+**Who is it for?**
+
+Developers using Workers.
+
+**Included with**
+
+Workers Subscription (free or paid).
```
Comment (pedrosousa) [inline products/fundamentals/src/content/notifications/notification-available.md]:
```suggestion
Workers subscription (free or paid).
```

File: products/fundamentals/src/content/notifications/notification-available.md [hunk 8]
```diff
@@ -357,81 +419,120 @@ Enterprise plans with paid add-on.
...
+
+**What should you do if you receive one?**
+
+No action is usually required. This notification gives users a high-level overview of their key workers’ metrics without checking the dashboard. Possible metrics include account usage and per-worker usage.
```
Comment (pedrosousa) [inline products/fundamentals/src/content/notifications/notification-available.md]:
```suggestion
No action is usually required. This notification gives users a high-level overview of their key Workers’ metrics without having to check the dashboard. Possible metrics include account usage and per-worker usage.
```
I'd capitalize "Workers", at least.

File: products/fundamentals/src/content/notifications/notification-available.md (hunk 1 above)
Comment (pedrosousa) [inline products/fundamentals/src/content/notifications/notification-available.md]: Nit: The notification description in the dashboard only mentions the failure case:
> The transferred DNSSEC zone is incorrectly configured.

**Review by pedrosousa — APPROVED** [review]

File: products/fundamentals/src/content/notifications/notification-available.md (hunk 1 above)
Comment (marciocloudflare) [reply products/fundamentals/src/content/notifications/notification-available.md]: Let me check this with the PM. Thanks!

**Pull request merged**